.gitignore

```diff
@@ -9,4 +9,5 @@ __pycache__
 *MRI_viewer.mp4
 /venv/
 xASL_GUI_run.sh
-*Thumbs.db
\ No newline at end of file
+*Thumbs.db
+Testing/testConfig.json
```

Comment:
Are you able to move this script outside ExploreASL at your side, before we know it you have a huge list for .gitignore ;) I would only put stuff here that are very general

Reply:
This is meant for the integration testing. Instead of having our paths hardcoded in the full bids pipeline workflow, everyone can have their own testConfig.json file. The benefit with the gitignore is that everyone can have their own and can change it without committing etc. You can check out the description in the integration testing script. If you still disagree, let's talk about it in the abba meeting.

Reply:
I have unresolved that. Not that I would disagree with Michael, but to give Henk the opportunity to resolve himself. Sorry - at least from my experience - if it is resolved, then I don't read the answer ;) Which would be a shame..

Reply:
Sure, you can checkout the following script `xASL_test_IntegrationTesting` and if you check out the header, there is a description:

```
% -----------------------------------------------------------------------------------------------------------------------------------------------------
% DESCRIPTION:  Integration testing script. Run the full pipeline on the flavor datasets using ExploreASL_Master.
%
% EXAMPLE:      [TestResults, CheckResults] = xASL_test_IntegrationTesting;
%
% Your testConfig.json could look like this e.g.:
%
% {
%    "pathExploreASL": "...\\ExploreASL",
%    "pathTest": "...\\FlavorTests",
%    "cmdCloneFlavors": "git clone git@github.com:ExploreASL/FlavorDatabase.git"
% }
%
% -----------------------------------------------------------------------------------------------------------------------------------------------------
% Copyright 2015-2021 ExploreASL
```